TLM/TLM/U/UIScaler.cs

```diff
@@ -14,7 +13,7 @@ public static class UIScaler {
         /// <summary>Caching because UIView.Instance.uiCamera can be null sometimes.</summary>
         private static float cachedGuiHeight = GUI_HEIGHT;
 
-        /// <summary>Screen width for GUI is always fixed at 1920.</summary>
+        /// <summary>Gets screen width for GUI (always fixed at 1920).</summary>
         public static float GuiWidth {
             // TODO: Double check if GUI never changes width, the code below can be a const
```

Comment:
This is not called. is this redundant?

Reply:
GuiWidth property is used from UResizer.Calculate
cachedGuiHeight is used below in GuiHeight property